File: example/flagship/net/SDK_V3/sample.cs
using Flagship.Main;

var visitor = Fs.NewVisitor("<VISITOR_ID>")
  .IsAuthenticated(true)
  .HasConsented(true)
  .WithContext(new Dictionary<string, object> {
    ["isVIP"] = true,
    ["country"] = "NL",
    ["loginProvider"] = "Google"
    })
  .Build();

  await visitor.FetchFlags();

var flag = visitor.GetFlag("btnColor", 'red');
var flag = visitor.GetFlag("btnSize", 13);
var flag = visitor.GetFlag("showBtn", true);

File: cmd/analyze/code-samples/flagship/php/SDK_V3/sample.php
use Flagship\Flagship;

Flagship::start("your_env_id", "your_api_key");
$visitor = Flagship::newVisitor("your_visitor_id")->build();
$visitor->updateContext("isVip", true);
$visitor->fetchFlags();

$flag = $visitor->getFlag("displayVipFeature", false);
$flag = $visitor->getFlag("vipFeatureSize", 13);
$flag = $visitor->getFlag("vipFeatureColor", "red");
